server/cmd/options/options.go

```diff
@@ -76,6 +81,20 @@ func NewOptions(rootDir string) *Options {
 		},
 	}
 
+	// Disable node related features to prevent the need for informers.
+	utilfeature.DefaultMutableFeatureGate.OverrideDefault(features.ServiceAccountTokenNodeBindingValidation, false)
+	utilfeature.DefaultMutableFeatureGate.OverrideDefault(features.ServiceAccountTokenNodeBinding, false)
+
+	factory := func(factory informers.SharedInformerFactory) serviceaccount.ServiceAccountTokenGetter {
+		return serviceaccountcontroller.NewGetterFromClient(
+			nil,
```

Comment:
The client is fallback if the informer fails. Wondering if the Optional factory we merged should accommodate that too?